test/parallel/test-crypto.js

```diff
@@ -141,3 +140,42 @@ assert.throws(function() {
 
 // Make sure memory isn't released before being returned
 console.log(crypto.randomBytes(16));
+
+/**
+ * Check if the stream function uses utf8 as a default encoding.
+ **/
+
+function testEncoding(options, assertionHash) {
+  const hash = crypto.createHash('sha256', options);
+  let hashValue = '';
+
+  hash.on('data', (data) => {
+    hashValue += data.toString('hex');
+  });
+
+  hash.on('end', () => {
+    assert.equal(hashValue, assertionHash);
+  });
+
+  hash.write('öäü');
+  hash.end();
+}
+
+// Hash of "öäü" in utf8 format
+const assertionHashUtf8 =
+  '4f53d15bee524f082380e6d7247cc541e7cb0d10c64efdcc935ceeb1e7ea345c';
+
+// Hash of "öäü" in ascii format
```

Comment:
@lmoe yeah, I think you got what I was trying to say. That you see the Utf8 part too is just github’s UI, usually the PR comments just apply to single lines (so, yeah, I’m talking about the part which is currently labelled `ascii`)